bn_mp_log_u32.c

```diff
@@ -0,0 +1,174 @@
+#include "tommath_private.h"
+#ifdef BN_MP_LOG_U32_C
+/* LibTomMath, multiple-precision integer library -- Tom St Denis */
+/* SPDX-License-Identifier: Unlicense */
+
+/* Compute log_{base}(a) */
+static mp_word s_pow(mp_word base, mp_word exponent)
```

Comment:
This file is called mp_log_u32.c in develop

Reply:
Ah, great, knew that this large renaming would get me at some point! ;-)